### ISSUE TITLE
Flow can't find 'chai' export expect()/assert()/should() 

### ISSUE THREAD

**Issue by gkedge** [flowtype/flow-typed#482]
@samwgoldman if I leverage chai using the customary import for 'expect':

```
import { expect } from 'chai';
```

Flow(v0.35.0) carps:

```
/reusable/utilities/foo.spec.js:14
 14: import { expect } from 'chai';
              ^^^^^^ Named import from module `chai`. This module has no named export called `expect`.
```

I am leveraging your flow-typed/definitions/npm/chai_v3.5.x/flow_v0.24.0-/chai_v3.5.x.js.

Do I need to add something to that declaration to expose 'expect' or import 'expect' differently?

**Comment by ryyppy** [comment]
I have the feeling that flow treats this libdef as a CommonJS module... can you check if you can do:

```javascript
import chai from 'chai';

const { expect } = chai;
```

If this works, then we need to declare the module exports differently

**Comment by gkedge** [comment]
@ryyppy I have some good, yet unsatisfactory news.  I can't repeat my issue right now; flow isn't balking at this any longer:
```
import { expect } from 'chai';
```

The 'unsatisfactory' part is that prior to attempting your suggestion, I did some `npm install`.  One of the items I took was flow-bin@0.36.0.  I went back to flow-bin@0.35.0 to see if I could reproduce the issue, but I can't. I'm gonna close this until it happens again.  Thanks for the quick turn around on that.

**Comment by ryyppy** [comment]
I guess that's good for now ✨

**Comment by flegall** [comment]
Hi, I've just hit the same bug today.

I've pushed a branch of a project that exposes it : https://github.com/flegall/monorepo-fullstack-starter/tree/flowtype_482

What's strange is that only the expect function has this problem in my case.
I'm using Jest with Chai Assertions. I've noticed that the behaviour doesn't appear any more when removing the jest definition file (which exposes a global expect method).
